frontend/src/components/NewRunParametersV2.tsx

```diff
@@ -229,8 +257,23 @@ class ParamEditor extends React.Component<ParamEditorProps, ParamEditorState> {
 
     const onClick = () => {
       if (this.state.isInJsonForm) {
+        let paramType = param.type;
+        let displayValue;
+        switch (paramType) {
+          case ParameterType_ParameterTypeEnum.LIST:
+            displayValue = JSON.parse(param.value || '[]');
```

Comment:
What happen if `JSON.parse` throws error?

Reply:
I think only 2 scenario will cause in valid JSON form
1. Wrong default value - pre check by the python compiler. 
2. Incorrect user input - will display error message (in next PR)